virtualizarr/tests/test_integration.py

```diff
@@ -11,3 +13,24 @@ def test_open_scalar_variable(tmpdir):
 
     vds = open_virtual_dataset(f"{tmpdir}/scalar.nc")
     assert vds["a"].shape == ()
+
+
+@pytest.mark.parametrize("format", ["json", "parquet"])
+def test_kerchunk_roundtrip(tmpdir, format):
+    # set up example xarray dataset
+    ds = xr.tutorial.open_dataset("air_temperature", decode_times=False)
+
+    # save it to disk as netCDF (in temporary directory)
+    ds.to_netcdf(f"{tmpdir}/air.nc")
+
+    # use open_virtual_dataset to read it as references
+    vds = open_virtual_dataset(f"{tmpdir}/air.nc")
```

Comment:
```suggestion
    vds = open_virtual_dataset(f"{tmpdir}/air.nc", indexes={})
```
This shouldn't matter but better to be explicit